DSCResources/MSFT_xCredSSP/MSFT_xCredSSP.psm1

```diff
@@ -203,12 +208,23 @@ function Set-TargetResource
 
                         foreach($DelegateComputer in $DelegateComputers)
                         {
-                            if(!$CurrentDelegateComputers.Contains($DelegateComputer))
+                            if($CurrentDelegateComputers -eq $NULL)
```

Comment:
These two cases can be combined with a single or operator. No else statement is needed. Like this:
```powershell
if (($CurrentDelegateComputers -eq $null) -or (!$CurrentDelegateComputers.Contains($DelegateComputer)))
{
...
}
```
The null expression will be evaluated first and the second expression will not run is $CurrentDelegateComputers is null.